p/api/fever-press.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Fever API for FreshRSS
+ * Version 0.1
+ * Author: Kevin Papst / https://github.com/kevinpapst
+ *
+ * API endpoint for the Press Android client (only tested with 1.5.4)
+ */
+
+function createFeverApiInstance()
+{
+	if (!class_exists('FeverAPI_PressClient', false)) {
+		class FeverAPI_PressClient extends FeverAPI
+		{
+			protected function getDaoForEntries()
+			{
+				return new FeverAPI_EntryDAO_PressClient();
+			}
+			protected function setGroupAsRead($id, $before)
+			{
+				$before = $this->convertBeforeToId($before);
+				$dao = $this->getDaoForEntries();
+				if ($id === 0) {
+					$dao = $this->getDaoForEntries();
+					$counts = $dao->countFever();
+					$dao->markReadEntries($counts['max']);
+				} else {
+					$dao->markReadCat($id, $before);
+				}
+			}
+		}
+
+		class FeverAPI_EntryDAO_PressClient extends FeverAPI_EntryDAO
+		{
+			protected function getSelectLimit($max_id, $since_id)
+			{
+				if (!empty($entry_ids)) {
+					return ' LIMIT 50';
+				} else if (!empty($max_id)) {
+					return ' LIMIT 50';
+				}
+
+				if ($since_id < 0) {
+					$since_id = 0;
+				}
+
+				return ' LIMIT '.intval($since_id).', 50';
```

Comment:
This syntax is MySQL-specific. I am not sure we will keep this file though, so no hurry in adding SQLite + PostgreSQL.

Reply:
I only added it, as many requests by Android users where pointing to Press. I don't need it and for Android there seem to be much more/better clients with greader support, so I don't mind if we remove it.